src/prefect/engine/state.py

```diff
@@ -467,6 +476,32 @@ class Finished(State):
     color = "#003ccb"
 
 
+class Looped(Finished):
```

Comment:
Makes me wonder if both `Looped` and `Mapped` should become `Looping` and `Mapping` since they imply that more work is going to be done

Reply:
Yea I thought about this; I'm happy to change it but here's my reasoning for `Looped`: when a Task enters a `Looped` state it let's us know that a single iteration of the Loop was complete.  The full "looping" mechanism actually involves `Looped -> Running -> Looped -> Running`.

This is in contrast to a `Running` state - when a Task is in a `Running` state, it actually means there is work currently being done.

Reply:
I'm in